src/encoded/types/base.py

```diff
@@ -308,7 +309,7 @@ class Item(snovault.Item):
         # publication
         'published': ALLOW_CURRENT,
         # experiment sets
-        'pre-release': ALLOW_LAB_VIEW_ADMIN_EDIT
+        'pre-release': ALLOW_VIEWING_GROUP_VIEW,
```

Comment:
It will not - actually every ACL should have SUBMITTER_CREATE - which (at least to my understanding) is the permission used when the create button is activated from the Item view (as opposed to the collection view, which has it's own ACL added via the Collection Class)